models/SDSC_ISG-TrendModel/main.R

```diff
@@ -0,0 +1,15 @@
+# Pull SDSC_ISG-TrendModel from repo
```

Comment:
If you're going to use an R script, please also provide an renv lockfile.

Otherwise, in this relatively simple case, you could get away with a bash script. See for example [the script](https://github.com/epiforecasts/europe-covid-forecast/blob/3d6245ad3a0b6c801555f571ddf1d5c2487a8509/main.sh) we have for EpiNow2. Using bash will result in much faster runs.